src/app.js

```diff
@@ -1 +1,19 @@
-'use strict';
+"use strict";
+const interval = 1000;
+const fs = require('fs');
+
+
+const fileCreator = () => {
+  try {
+    const date = new Date();
+    const timestamp = `${date.getHours}_${date.getMinutes}_${date.getSeconds}`;
+    fs.writeFile(`app-${timestamp}.log`, timestamp, (err) => {
+      if (err) throw err;
```

Comment:
you are already catching error, using try-catch. just handle it in catch(error) block, not needed to do it here